cli/internal/cmd/iam.go

```diff
@@ -209,6 +222,7 @@ func (c *iamCreator) create(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	c.log.Debugf("IAM create command generated iamIdFile")
```

Comment:
I think you're confusing this here. (easily happens because the naming is very similar across the levels of abstraction here)

the `Create` method has an output of the data type `iamid.File` which has been chosen to stay consistent with the inner workings of the `constellation create` command
For reference:
https://github.com/edgelesssys/constellation/blob/ea52aafcd32ea3ac01f874aad3f25b8e6e0dad0a/cli/internal/cmd/create.go#L144-L146

Reply:
Ah I see. Yes, I think that `generated` in the same sentence with `file` reads for me as `file created`. The generation of the struct `iamid.File` itself is not very likely to fail and only a minor part of the `Create` method. Therefore I wouldn't mention it in this file. In contrast, the creation of the cloud resources is the what is "likely" or one might be interested when this step has been completed. 

Yes, go ahead and change it in both files.